packages/runtime/container-runtime/api-report/container-runtime.legacy.alpha.api.md

```diff
@@ -100,6 +100,7 @@ export interface ICompressionRuntimeOptions {
 // @alpha
 export interface IContainerRuntimeOptions {
     readonly chunkSizeInBytes?: number;
+    readonly compatibilityMode?: "1" | "2";
```

Comment:
In practice, this would be "min supported version for cross-client compatibility", right? Then I would call it something like `minVersionForCollaborators`.  Or I guess it'll take a semver range, so `collaboratorsVersionRangeRequirement`.  And would we actually use something like the semver Range type, rather than string?

For this prototype PR I'd go all the way to the long-term vision, I don't know how "1" and "2" will fit into that world so not sure what to make of them here. Maybe it'll become clear later in the PR.

Reply:
Yeah we can consider other names like the ones you suggested. I initially chose `compatibilityMode` to stay consistent with the declarative model.